### ISSUE TITLE
[ Suggestion ] Use discordid to represent your user profile

### ISSUE THREAD

**Issue by taichikuji** [m0nk3ybraindead/m0nk3ybraindead#1]
My suggestion is that instead of linking a discord group, you might want to, instead, just link your user through discordid, which means you don't need to create a discord group just for people to have a way to talk with you, they can just DM you.

If you are interested, you can check the guide on how to get your discordid link ready here: [[ DiscordID Wiki ](https://github.com/taichikuji/discordid/wiki)]

TLDR;
Suggestion to replace this:
```markdown
- 📫 How to reach me: [`Discord`](https://discord.gg/tRMQcwhJcC)
```
With this:
```markdown
- 📫 How to reach me: [`Discord`](https://discordid.netlify.app/?id=843803596554633246)
```

**Comment by lrnzdc3696** [comment]
No, because I use many accounts so it's better to join a guild with most of my testing accounts
The accounts are for testing nothing

**Comment by taichikuji** [comment]
Okay, cool! Was just a suggestion. Thanks for the quick response tho! I'll close the issue then.
